### DISCUSSION TITLE
Error when creating new menu item: attempt to read property "language"; Mega Menu button no longer showing

### DISCUSSION THREAD

**Discussion by jlilio** [templaza/astroid-framework#259] [Q&A]
Development Environment:
- XAMPP for Windows 8.0.23
- Joomla 4.2.5
- Astroid Framework Version: 2.6.3
- Template Version: 2.6.3 (One)

Error when creating new menu on the back end administration:
Warning: Attempt to read property "language" on null in <path>\libraries\astroid\framework\layouts\fields\astroidmenuoptions.php on line 62

Effect: When creating a new menu, Astroid Menu Options > Mega Menu button is no longer showing. Downgrading to framework 2.6.2 did not correct the error. All previously created menus are fine (i.e., no error message pertaining to language and mega menu is still available).

This may be a bug. Any ideas?

![20221208-issue-astroidmenuoptions-line-62-attempt-to-read-property-language-error-megamenu-missing](https://user-images.githubusercontent.com/6914851/206367779-ce25b1b8-67f0-4fe1-9f8e-09b446dc6db5.png)

**Comment by Chacapamac** [comment]
I just try
Joomla 4.2.6
Framework & Template One —> v2.6.3
Live server: php 8.1.13

I Create a new menu without problems?

I will try on my local Mamp server the same and come back

— Did you try to re-install Framework/template 2.6.3, clear all caches and recreate the menu?
— What Joomla type of menu is yours? — Mine is a “Single Article” menu
